.github/ISSUE_TEMPLATE/feature.md

```diff
@@ -0,0 +1,19 @@
+---
+name: Feature Request
+about: Please first, look at existing issues to see if the feature has been requested before.
+labels: enhancement
+---
+Please first, look at [existing issues](https://github.com/openzipkin/zipkin-ruby/issues) to see if the feature has been requested before. If you don't find anything tell us what problem you’re trying to solve. Often a solution already exists! Don’t send pull requests to implement new features without first getting our support. Sometimes we leave features out on purpose to keep the project small.
+
+**Feature:**
```

Comment:
how about having the same formatting as in the bug.md? e.g. use markdown headers instead of bold.